### ISSUE TITLE
Radiation Intensity display stuck at 0%

### ISSUE THREAD

**Issue by RenaKunisaki** [budak7273/ArmorModules#32]
## Describe the bug

The Radiation Intensity reading from the Integrated Hazmat Suit always says 0%.

## To Reproduce

1. Equip the Integrated Hazmat Suit and ensure it has filters
2. Craft an Unstable Somersloop (or probably pick up any radioactive item) and keep it in your inventory

## Expected behavior

Radiation Intensity display should increase. Instead it stays at 0%, even though the filter is depleting and the vanilla Radiation Level display is shown.

## Multiplayer?

Haven't tried multiplayer.

## Screenshots or Videos

Can add screenshot if needed.

## Other Mods

```
Mod Name                         Mod Reference                Version
Any Route For Any Vehicle        UniversalVehiclePaths        1.0.3
Blocks                           Blocks                       3.0.5
Circuitry - Logic gates & More   FicsitWiremod                0.12.20
Community Basic Tracks           CommunityBasicTracks         1.0.2
Community Railway Set            CommunityRailwaySet          1.0.4
Community Road Network           CommunityRoadNetwork         1.0.3
Dependency To Blueprint Download FileSystemLibrary            1.0.1
DI Common Lib                    DI_Common                    3.8.0
DI FunctionsLibrary              DI_FunctionsLibrary          3.8.0
DI More Lights                   DI_More_Lights               3.8.0
DI Power Indicators              DI_Power_Indicators          3.8.0
Factory Props                    Factory_Prop_Mod             1.3.0
Fire Smoke and Water             FireSmokeWater               1.0.2
Hotterbars - Hotbar keybinds     KeybindableHotbar            1.0.1
Infinite Nudge                   InfiniteNudge                1.1.0
Infinite Zoop                    InfiniteZoop                 1.6.8
Item Spawner                     ItemSpawner                  2.0.0
Linear Motion                    LinearMotion                 2.0.24
MAM Enhancer                     MAMTips                      1.3.1
MiniMap                          MiniMap                      1.0.13
Modpack: All PowerSuit Modules   ArmorModules__Modpack_All    2.2.0
No Early Access Mark             NoEarlyAccess                1.0.3
No New Label                     NoNewLabel                   1.0.1
No Portable Miners - Redux       NPMRedux                     1.0.2
Organized Crates                 OrganizedCrates              2.0.4
PowerSuit                        PowerSuit                    1.3.0
PowerSuit Logistic Modules       LogisticModules              1.1.0
PowerSuit Modules                ArmorModules                 4.2.0
PSM: Flight                      ArmorModules_Flight          2.1.0
PSM: Fresh from R&D              ArmorModules_Special         2.1.0
PSM: Mobility                    ArmorModules_Mobility        2.2.0
PSM: Pioneer Protection          ArmorModules_Defense         2.1.0
PSM: Power Generation            ArmorModules_PowerGeneration 2.2.0
PSM: Suit Regulation             ArmorModules_SuitRegulation  2.1.0
Skin It - Five Vehicle Skins     VehiclesVariety              1.0.1
Speedometer                      Velocimeter                  2.0.2
Structural Solutions             SS_Mod                       1.1.22
TFIT - Ficsit Information Tool   TFIT                         2.2.0
Transformers Explorer            TransformersExplorer         1.0.3
Unrestricted Icons               UnrestrictedIcons            1.0.1
Upside Down Foundations More     UpsideDownFoundations        1.0.9
Vehicle Path Viewer              ShowSplinePath               1.1.2
Zip-Strips                       ZipStrips                    1.0.4
```

## Additional context

Playing on Artix Linux, if that matters.

**Comment by budak7273** [comment]
I think this is due to the Update 8 changes that affect how widgets in the player's HUD need to be implemented. I'll take a look at it this evening. Thanks for the report

**Comment by budak7273** [comment]
Fixed in v4.2.1
